keybase/platform_windows.go

```diff
@@ -409,3 +414,38 @@ func (c context) GetAppStatePath() string {
 func (c context) IsCheckCommand() bool {
 	return c.isCheckCommand
 }
+
+// copied from watchdog
+func (c context) stopKeybaseProcesses() error {
+	path, err := Dir("Keybase")
+	if err != nil {
+		c.log.Infof("Error getting Keybase directory: %s", err.Error())
+		return err
+	}
+
+	c.runKeybase(false)
```

Comment:
sorry, that was leftover from my more experimental branch, it doesn't have to go in like this now if it is confusing.

The idea was to have the regular windows installer put most target files in an `update` subdirectory, then make the updater smart enough to look there and do the copying if available. The name of the directory and the name of the new updater would be written to the registry on install for the old updater to find. Then the startup shortcuts would invoke the updater with a launch command - it would go like this, with old updater A and new updater B in the update subdirectory:
- launch Keybase by invoking A -launch
- A looks in registry and finds updater B in the `update` subdir
- (if no update present, it just launches Keybase normally)
- A double checks that all processes are stopped, then invokes `update\B.exe -launchUpdate`, and quits
- B starts and knows it has to move everything up. After that, in invokes the copy of B in the target directory with `-launchUpdated`, then exits
- B starts with `-launchUpdated', telling it to cleanup and then launch the new Keybase

sorry that's a digression from this PR but I wanted to discuss sometime.
The advantage is that the updater only invokes or removes the version it was built with, we have room for future changes, and the next step of getting rid of MS install altogether would be easier with this. Downside is that it is scary! how do you revert? do you keep a copy of the entire old installation just in case? (it's like 180MB) etc.